app/src/main/java/com/mathewsachin/fategrandautomata/ui/prefs/MoreSettingsFragment.kt

```diff
@@ -57,15 +57,6 @@ class MoreSettingsFragment : PreferenceFragmentCompat() {
 
             it.summary = storageProvider.rootDirName
         }
-
```

Comment:
In this same file, I noticed that the lines 76-84 were also introduced with 92b9ae71, but I'm not sure what they are doing. So I left them there. 😿

Reply:
`storageProvider.rootDirName` is a bad naming choice from my side. I meant the topmost directory the user picked for storing files here, which can easily be confused with super-user (root).

76-84 are disabling the In-built recording feature when using `Root for Screenshots`.
The in-built recorder uses the same `MediaProjection` used for screenshots to perform the recording, which won't work when using Root, so it is disabled.

Reply:
When I did my youtube guide, I recorded the FGO gameplay using root and recorded my voice using MediaProjection.

So recording with root definitely worked back then.

Reply:
Oh, I didn't mind `rootDirName`, actually. I wanted to comment lines 76-84, but GitHub limits the scope of the reviews to the parts of the file that were changed only. Thanks for the explanation :)